packages/types/src/interfaces/parachains/definitions.ts

```diff
@@ -42,32 +42,51 @@ export default {
       validatorIndices: 'BitVec'
     },
     AuctionIndex: 'u32',
+    BackedCandidate: {
+      candidate: 'CommittedCandidateReceipt',
+      validityVotes: 'Vec<ValidityAttestation>',
+      validatorIndices: 'BitVec'
+    },
     Bidder: {
       _enum: {
         New: 'NewBidder',
         Existing: 'ParaId'
       }
     },
     CandidateCommitments: {
-      fees: 'Balance',
       upwardMessages: 'Vec<UpwardMessage>',
-      erasureRoot: 'Hash',
+      horizontalMessages: 'Vec<OutboundHrmpMessage>',
       newValidationCode: 'Option<ValidationCode>',
-      processedDownwardMessages: 'u32'
+      headData: 'HeadData',
+      processedDownwardMessages: 'u32',
+      hrmpWatermark: 'BlockNumber'
     },
-    CandidateReceipt: {
-      parachainIndex: 'ParaId',
+    CandidateDescriptor: {
+      paraId: 'u32',
```

Comment:
Should this be `paraId: 'ParaId',?

Reply:
And same thing below in HrmpChannelId

Reply:
I saw that, copied it as supplied - will do a proper once-over once it actually lands properly to align 100% with Polkadot.